pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -250,7 +252,8 @@ def use_dp(self) -> bool:
     def use_ddp(self) -> bool:
         return self._distrib_type in (
             DistributedType.DDP, DistributedType.DDP_SPAWN, DistributedType.DDP_SHARDED,
-            DistributedType.DDP_SHARDED_SPAWN, DistributedType.DEEPSPEED, DistributedType.TPU_SPAWN
+            DistributedType.DDP_SHARDED_SPAWN, DistributedType.FULLY_SHARDED, DistributedType.DEEPSPEED,
+            DistributedType.TPU_SPAWN
```

Comment:
it's unclear to me what `use_ddp` stands for at this point with all these distributed types supported

Reply:
I think the primary usage is knowing whether to use a distributed sampler, but this logic should ideally be re-written to be a property of the training type plugin. I was hoping to get to that in #6090